systemtest/src/main/resources/StrimziUpgradeST.json

```diff
@@ -111,9 +113,10 @@
       "continuousClientsMessages": 0
     },
     "environmentInfo": {
+      "k8sVersion": "1.15",
```

Comment:
I think this is confusing ... is this minimum or maximum version? I think it would be better to name this `maxK8sVersion`.

Reply:
It's maximum version, going to change it now. Same for rest comments.